apps/fz_http/lib/fz_http_web/templates/root/auth.html.heex

```diff
@@ -9,9 +9,10 @@
 
   <%= for {provider, config} <- @openid_connect_providers do %>
     <p>
-      <%= link(
+      <%=
+        link(
           "Sign in with #{config[:label]}",
-          to: authorization_uri(@openid_connect, provider),
+          to: "/auth/oidc/#{provider}",
```

Comment:
Can you use a route helper here? Check `mix phx.routes` to get a list of all the route helpers.

Haven't tested, but it would probably be

```elixir
Routes.auth_oidc_path(conn, :redirect_oidc_auth_uri)
```

Reply:
See https://hexdocs.pm/phoenix/Phoenix.Router.html#module-helpers